implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogConfig.java

```diff
@@ -76,4 +76,8 @@ default Validated<?> validate() {
                 checkRequired("uri", DatadogConfig::uri));
     }
 
+    default boolean publishAverage() {
```

Comment:
See https://github.com/micrometer-metrics/micrometer/issues/6844#issuecomment-3549762243, this should work for all the registries that publish avg.